src/commands/attest/create.ts

```diff
@@ -67,7 +67,7 @@ export default class CreateAttestations extends Command {
         for (const profile of evaluation.profiles) {
           for (const control of profile.controls) {
             controls[control.id] = control
-            search.addText(control.id, control.id + ': ' + (control.title || '') + ' ' + control.desc || '')
+            search.addText(control.id, `${control.id} ${control.title} ${control.desc}`)
```

Comment:
not a match with the previous behavior (we're losing the colon there) and also doesn't address the issue of what happens if one of those attributes is missing
```suggestion
            search.addText(control.id, `${control.id}: ${control.title || ''} ${control.desc || ''}`)
```